tests/test_database.py

```diff
@@ -82,15 +77,27 @@ async def read_note(request):
     return JSONResponse(content)
 
 
-@app.route("/notes/{note_id:int}/text", methods=["GET"])
-async def read_note_text(request):
+async def read_note_text(request: Request):
     note_id = request.path_params["note_id"]
     query = sqlalchemy.select([notes.c.text]).where(notes.c.id == note_id)
     result = await database.fetch_one(query)
     assert result is not None
     return JSONResponse(result[0])
 
 
+app = Starlette(
+    routes=[
+        Route("/notes", endpoint=list_notes, methods=["GET"]),
+        Route("/notes", endpoint=add_note, methods=["POST"]),
+        Route("/notes/bulk_create", endpoint=bulk_create_notes, methods=["POST"]),
+        Route("/notes/{note_id:int}", endpoint=read_note, methods=["GET"]),
+        Route("/notes/{note_id:int}/text", endpoint=read_note_text, methods=["GET"]),
+    ],
+    on_startup=[startup],
+    on_shutdown=[shutdown],
+)
```

Comment:
😌 - Yes.

Reply:
This is what I like to see.

Reply:
So direct, so upfront, so relaxing.

Reply:
Exactly, you can view all routes/middlewares in one view. Much more readable.